### PR TITLE
Add simple unit test for TF 1.14 learning rate schedules

### LINKED ISSUE #106: Add test for Bop with tf.keras.optimizer.schedules (TF 1.14+)
_Originally posted by @lgeiger in https://github.com/plumerai/larq/pull/103/files_

### PR DESCRIPTION
**Pull request by lgeiger**
Fixes #106

### CODE REVIEW

**Comment by arashb** [comment]
@lgeiger s the default accuracy target (0.75) in _test_optimizer test conservative enough to be proper test for optimizer with this new hyperparameters?

File: larq/optimizers_test.py [hunk 1]
```diff
@@ -104,6 +104,24 @@ def test_bop_accuracy(self):
...
+                    3.0, decay_steps=1.0, decay_rate=0.5
+                ),
+                gamma=tf.keras.optimizers.schedules.InverseTimeDecay(
+                    3.0, decay_steps=1.0, decay_rate=0.5
+                ),
+                fp_optimizer=tf.keras.optimizers.Adam(0.01),
+            ),
+            test_kernels_are_binary=True,
+        )
+
```
Comment (arashb) [inline larq/optimizers_test.py]: is the default accuracy target (0.75) in _test_optimizer test conservative enough to be proper test for optimizer with this new hyperparameters?

Comment (lgeiger) [reply larq/optimizers_test.py]: Accuracy is above 90% so it should be fine.
